coderd/externalauth/externalauth.go

```diff
@@ -493,8 +494,27 @@ func ConvertConfig(entries []codersdk.ExternalAuthConfig, accessURL *url.URL) ([
 }
 
 // applyDefaultsToConfig applies defaults to the config entry.
-func applyDefaultsToConfig(config *codersdk.ExternalAuthConfig) {
-	defaults := defaults[codersdk.EnhancedExternalAuthProvider(config.Type)]
+func configDefaults(config *codersdk.ExternalAuthConfig) {
```

Comment:
Why shorten the name? `configDefaults` seems like it could return defaults, whereas apply states that it'd apply to the arg.

Reply:
Renamed. I just had to make another func that actually applies the defaults. See new names of `applyDefaultsToConfig` and `copyDefaultSettings(config *codersdk.ExternalAuthConfig, defaults codersdk.ExternalAuthConfig)`